hybridse/src/udf/udf.cc

```diff
@@ -836,6 +836,19 @@ void ucase(codec::StringRef *str, codec::StringRef *output, bool *is_null_ptr) {
     *is_null_ptr = false;
 }
 
+void reverse(codec::StringRef *str, codec::StringRef *output, bool *is_null_ptr) {
+    if (str == nullptr || str->size_ == 0 || output == nullptr || is_null_ptr == nullptr) {
```

Comment:
Returned strref not modified if str size == 0 ?

Reply:
Done.